Geometry/HcalCommonData/src/HcalDDDRecConstants.cc

```diff
@@ -448,29 +468,28 @@ HcalDDDRecConstants::HcalCellTypes(HcalSubdetector subdet) const {
     int isub   = (subdet == HcalBarrel) ? 0 : 1;
     std::vector<HcalDDDRecConstants::HcalEtaBin> etabins = getEtaBins(isub);
     std::vector<int> missPhi;
-    for (unsigned int bin=0; bin<etabins.size(); ++bin) {
+    for (auto etabin : etabins) {
```

Comment:
It was because some of the function calls were not declared as const. This is done now